packages/vue/src/components/range/MultiRange.jsx

```diff
@@ -0,0 +1,252 @@
+import { Actions, helper } from '@appbaseio/reactivecore';
+import VueTypes from 'vue-types';
+import Title from '../../styles/Title';
+import Container from '../../styles/Container';
+import { UL, Checkbox } from '../../styles/FormControlList';
+import { connect } from '../../utils/index';
+import types from '../../utils/vueTypes';
+
+const {
+  addComponent,
+  removeComponent,
+  watchComponent,
+  updateQuery,
+  setQueryListener
+} = Actions;
+
+const { isEqual, checkValueChange, getClassName } = helper;
+const MultiRange = {
+  name: 'MultiRange',
+  data() {
+
+    this.state = {
+      currentValue: [],
+      showModal: false,
+      // selectedValues hold the selected items as keys for O(1) complexity
+      selectedValues: {},
+    };
+    this.type = 'range';
+    this.locked = false;
+    return this.state;
+  },
+  props: {
+
+    beforeValueChange: types.func,
+    className: VueTypes.string.def(''),
+    componentId: types.stringRequired,
+    customQuery: types.func,
+    data: types.data,
+    dataField: types.stringRequired,
+    defaultSelected: types.string,
+    filterLabel: types.string,
+    innerClass: types.style,
+    react: types.react,
+    showFilter: VueTypes.bool.def(true),
+    showCheckbox: VueTypes.bool.def(true),
+    title: types.title,
+    URLParams: VueTypes.bool.def(false)
+  },
+  methods: {
+    setReact(props) {
+      if (props.react) {
+        props.watchComponent(props.componentId, props.react);
```

Comment:
You won't be able to find `watchComponent` in props.

Reply:
changed it to `this`